### ISSUE TITLE
allow vm args to be specified

### ISSUE THREAD

**Issue by Andersmholmgren** [google/test_runner.dart#11]
I'm using async and enums in my test code and need to be able to specify flags to dart to enable them

**Comment by nicolasgarnier** [comment]
You're right that we need to have this capability. One of the issue is that we are currently running VM tests with `pub run` and it's not possible to pass flags to the underlying VM using `pub run` but we are planning on moving away from `pub run` and using `dart` directly instead. We loose automatic transformers but get rid of a bunch of issues with `pub run` (forced checked mode, no ability to gather code coverage data, Platform.script.path not working etc...).
